docs/style-properties.md

```diff
@@ -99,7 +99,7 @@ stylebox-source: "images/stylebox.png"
 <!-- @property-type=$val -->
 Size type representing `bevy::prelude::Val` type. Possible values:
 - `auto` for `Val::Auto`
-- `undefined` for `Val::Undefined`
+- `undefined` for `Val::Px(0.)`
```

Comment:
This is kinda weird, right?

Reply:
Yeah, if you need an `undefined` defined value, then it should probably map to `Val::Auto`.